ts/Extensions/Annotations/Types/BasicAnnotation.ts

```diff
@@ -250,6 +237,13 @@ interface BasicAnnotation {
     defaultOptions: Annotation['defaultOptions'];
     basicType: string;
 }
+namespace BasicAnnotation {
+    export interface AnnotationBasicControlPoints {
```

Comment:
Please rename it to type `ControlPoints` (so full name: `BasicAnnotation.ControlPoints`).